google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOpenTelemetryOptions.java

```diff
@@ -75,7 +75,7 @@ public FirestoreOpenTelemetryOptions build() {
      */
     @Nonnull
     public FirestoreOpenTelemetryOptions.Builder setTracingEnabled(boolean enable) {
```

Comment:
nit - maybe rename the param too for consistency

Reply:
SG! Done.